components/ManagerSubmitButton.tsx

```diff
@@ -0,0 +1,54 @@
+import urlsForRouter from "helpers/urlsForRouter";
+import { Button } from "lbh-frontend-react";
+import { NextRouter, useRouter } from "next/router";
+import PropTypes from "prop-types";
+import React from "react";
+import PageSlugs, { urlObjectForSlug } from "steps/PageSlugs";
+
+interface Props {
+  onSubmit(router: NextRouter): Promise<void>;
+  buttonText: string;
```

Comment:
Generally, prefer `children` when specifying a prop that's used as the child of the main element of the component. This could be any React node.

Reply:
Good to know - would we still want to do this if we're passing the "status" through as an enum?

Reply:
No. This was a note for future reference. With status, it's definitely a normal property.